eisbuk-admin/src/components/atoms/IntervalCard/__testData__/dummyData.ts

```diff
@@ -9,12 +9,23 @@ import { luxonToFB } from "@/utils/date";
 
 const date = luxonToFB(DateTime.fromISO(__storybookDate__));
 
-export const baseProps: Props = {
+export const bookedSlot: Props = {
   date,
   type: SlotType.Ice,
   interval: {
     startTime: "09:00",
     endTime: "10:00",
   },
   notes: "",
+  booked: false,
+};
+export const nonBookedSlot: Props = {
+  date,
+  type: SlotType.Ice,
+  interval: {
+    startTime: "10:00",
+    endTime: "11:00",
+  },
+  notes: "",
+  booked: true,
```

Comment:
There's no need copy paste all. In the tests you can just use:
```jsx
// for booked variant test
render(<IntervalCard {...baseProps} booked />)

// for not booked variant test
render(<IntervalCard {...baseProps} />)
```
Everything else is the same, except for interval property (which can be the same for both tests)

Reply:
This is also the beauty of optional boolean flag we talked about on some of the previous PRs. In this case it's `booked` and the sole presence/absence of `booked` in the JSX will produce truthy/falsy value. The only scenario where you would need to explicitly set the `booked` property would be if you're using the condition, i.e.
```jsx
<IntervalCard {...someProps} booked={someCondition} />
```